test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -485,8 +494,10 @@ private void wipeCluster() throws Exception {
             waitForPendingRollupTasks();
         }
 
-        // Clean up SLM policies before trying to wipe snapshots so that no new ones get started by SLM after wiping
-        deleteAllSLMPolicies();
+        if(preserveSLMPoliciesUponCompletion() == false) {
```

Comment:
Super minor nit:

```suggestion
        if (preserveSLMPoliciesUponCompletion() == false) {
```

Reply:
would it make sense to include this in the checkstyle rules? (ie. `WhitespaceAround`)

Reply:
Changes the checkstyle rules can be pretty invasive across the board (and we don't want one-off rules). I think @pugnascotia is looking into a single commit reformatting, maybe this rule should be considered for that effort ?

Reply:
The formatting effort also introduces checking of the formatting, so we won't need to have specific rules for this sort of this.